src/VisualStudio/Core/Impl/Options/Style/AbstractCodeStyleOptionViewModel.cs

```diff
@@ -56,6 +57,7 @@ public virtual NotificationOptionViewModel SelectedNotificationPreference
             Preferences = preferences ?? GetDefaultPreferences();
             NotificationPreferences = notificationPreferences ?? GetDefaultNotifications();
             GroupName = groupName;
+            GroupNameAndDescription = $"{groupName}, {description}";
```

Comment:
We were getting around concatenating these before in a seriously ugly way. I can explain if needed. This is much better.